featuretools/primitives/transform_primitive.py

```diff
@@ -336,6 +337,30 @@ class Weekday(DatetimeUnitBasePrimitive):
     name = "weekday"
 
 
+class NumCharacter(TransformPrimitive):
```

Comment:
Should we name this `NumCharacters` to be consistent with `NumWords` below?

Reply:
Changed